src/packet/header/standard_header.rs

```diff
@@ -72,28 +108,51 @@ impl HeaderReader for StandardHeader {
 
 #[cfg(test)]
 mod tests {
-    use crate::infrastructure::DeliveryMethod;
+    use crate::net::constants::STANDARD_HEADER_SIZE;
     use crate::packet::header::{HeaderReader, HeaderWriter, StandardHeader};
-    use crate::packet::PacketTypeId;
-    use crate::protocol_version::ProtocolVersion;
+    use crate::packet::{DeliveryGuarantee, EnumConverter, OrderingGuarantee, PacketType};
     use std::io::Cursor;
 
     #[test]
-    pub fn serializes_deserialize_packet_header_test() {
-        let packet_header = StandardHeader::default();
-        let mut buffer = Vec::with_capacity((packet_header.size() + 1) as usize);
+    fn serialize() {
+        let mut buffer = Vec::new();
+        let header = StandardHeader::new(
+            DeliveryGuarantee::Unreliable,
+            OrderingGuarantee::Sequenced(None),
+            PacketType::Packet,
+        );
+        header.parse(&mut buffer).is_ok();
+
+        // [0 .. 3] protocol version
```

Comment:
Not sure what this comment means.

Reply:
I skipped the first 4 bytes who are the CRC32, however, I can't easily hardcode those into the `vec!` above it, simply because I don't know the CRC32 bytes yet. So I decided to just skip those bytes and start asserting on the other bytes.